lib/vm/src/cache.rs

```diff
@@ -119,7 +119,7 @@ mod test {
     use cosmwasm::mock::{dependencies, mock_params, MockApi, MockStorage};
     use cosmwasm::types::coin;
 
-    static CONTRACT: &[u8] = include_bytes!("../testdata/contract.wasm");
+    static CONTRACT_0_7: &[u8] = include_bytes!("../testdata/contract_0.7.wasm");
```

Comment:
Why not just overwrite the other one?
I guess to test compatibility?

Reply:
For some tests the latest contract is more appropriate, for some an explicit version. I made everything explicit here, which might not be a perfect solution long term. There is a `contract.wasm` prepared that is a symbolic link to the latest versioned file name.